tests/obc/test_spectral.py

```diff
@@ -101,12 +100,12 @@ def test_memoizer(
     spectral = OBCMemoizer(spectral)
     a_ji, a_ii, a_ij = _make_periodic(a_xx, block_sections)
     x_ii = spectral(a_ii=a_ii, a_ij=a_ij, a_ji=a_ji, contact=contact)
-    assert np.allclose(x_ii, xp.linalg.inv(a_ii - a_ji @ x_ii @ a_ij), atol=1e-5)
+    assert xp.allclose(x_ii, xp.linalg.inv(a_ii - a_ji @ x_ii @ a_ij), atol=1e-5)
```

Comment:
I'm not a huge fan of having the tolerances hard-coded here, I guess we  should have all solvers convergence tol somewhere? Anyway not a big deal

Reply:
Definitely not a fan either. I think we talked about this already at some point but to test the Spectral OBC solver and the NEVP we should somehow construct matrices with hand-picked eigenvalues. That could maybe also solve this?

Alternatively, cranking up the refinement iterations always helps :sweat_smile:

Reply:
Yes, maybe open an issue for the eigenvalues solvers tests, I foresee @almaeder being very suited for this issue :smile:

Reply:
See #51